### PR TITLE
day 54 progress

### PR DESCRIPTION
**Pull request by nayabatir1**

### CODE REVIEW

File: atir-nayab/28-03-2022/pod.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- kubernetes pods are the foundational unit for all higher kubernetes objects.
+- a pod hosts one or more containers
+- it can be created using either a command or a yml/json file
+- use kubectl to create pods, view the running ones, modifying their configuration or terminate them. kubernetes will attempt to restart a failing pod by default.
+- if the pod fails to start indefinitely, we can use the kubectl describe command to know what went wrong
+
+## why does kubernetes use a pod as the smallest deployable unit, and not a single container?
+
+there are good reasons to add a layer of abstraction represented by the pod.  
+To manage a container, kubernetes needs additional information, such as a restart polucy, which defines what to do with a container when it terminates, or a liveness probe, which defines an action to detect if a process in a container is still alive from the applications perspective, such as web server responding to http requests.
```
Comment (kranurag7) [inline atir-nayab/28-03-2022/pod.md:14]:
```suggestion
To manage a container, kubernetes needs additional information, such as a restart policy, which defines what to do with a container when it terminates, or a liveness probe, which defines an action to detect if a process in a container is still alive from the applications perspective, such as web server responding to http requests.  
```

File: atir-nayab/28-03-2022/pod.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# deploying your first nginx pod
+
+## what are k8s pods?
+
+- kubernetes pods are the foundational unit for all higher kubernetes objects.
+- a pod hosts one or more containers
+- it can be created using either a command or a yml/json file
+- use kubectl to create pods, view the running ones, modifying their configuration or terminate them. kubernetes will attempt to restart a failing pod by default.
+- if the pod fails to start indefinitely, we can use the kubectl describe command to know what went wrong
```
Comment (kranurag7) [inline atir-nayab/28-03-2022/pod.md:9]:
```suggestion
- if the pod fails to start indefinitely, we can use the `kubectl describe pod <pod-name>` command to know what went wrong
```

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Comment by kranurag7** [comment]
Thank you @nayabatir1 for updating the changes. LGTM

**Comment by saiyam1814** [comment]
It's great!!

**Comment by kranurag7** [comment]
Yes @nayabatir1 I go through it. It's really good.

**Pull request merged**